docs/sources/auth/jwt.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "JWT Authentication"
+description = "Grafana JWT Authentication"
+keywords = ["grafana", "configuration", "documentation", "jwt", "jwk"]
+aliases = ["/docs/grafana/latest/installation/jwt/"]
+weight = 300
++++
+
+# JWT Authentication
+
+You can configure Grafana to accept a JWT token provided in the HTTP header. The token is verified using a public key(s) stored in a file or provided by the JWKS endpoint.
+You can specify some of the JSON fields embedded in JWT (called claims) to use either as a username or as an email to sign in.
```

Comment:
Unclear. Use either what as a username or email to sign in?

Reply:
I removed this sentence cause it's too heavyweight for an intro paragraph.
The whole username/email thing is described in the "Configure login claim" section so it's redundant to explain it in the intro.